### ISSUE TITLE
Are you still supporting this?

### ISSUE THREAD

**Issue by ghost** [ethanwillis/thriftbooksapi#1]
i tried to install it but got a missing dependency error. Are you still keeping this thirtbooks-api up to date?

**Comment by ethanwillis** [comment]
I haven't maintained this in a couple of years, but I wouldn't mind putting some time into it. Give me a couple days and I'll get back to you.

**Comment by ghost** [comment]
Hey Ethan, i contacted Thriftbooks support, and it seems they have their
own API. That would probably be the better way to go, do you agree?

Thanks

Guy

**Comment by ethanwillis** [comment]
@guyatyaletowninteractive  I agree. At the time I wrote this they didn't have an API available. So I would definitely recommend going that route!

**Comment by dylan-albertazzi** [comment]
@ethanwillis Do you know the web address of the thriftbooks api? I can't seem to find it.
